easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.6.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb

```diff
@@ -0,0 +1,41 @@
+name = 'TensorFlow'
+version = '1.6.0'
+cuda_ver = '9.1.85'
+versionsuffix = '-Python-%%(pyver)s-CUDA-%s' % cuda_ver
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
+    'TensorFlow-%(version)s_dont_expand_cudnn_path.patch',
+    'TensorFlow-%(version)s_dont_expand_cuda_path.patch',
+    'TensorFlow-%(version)s_remove-msse-hardcoding.patch',
+]
+checksums = [
+    '03cf1423446abbead6bd8c3cf6e6affa7d99746cd119691b012aac9a1795f4fb',  # v1.6.0.tar.gz
+    '198120971886fd66144675deb4fcaed902e7834ac8055363e19220f21626aaf5',  # TensorFlow-1.6.0_swig-env.patch
+    '5989d00feb3e78fdadb6c118f2991e96018880f81b5201b7e957fa300df26160',  # TensorFlow-1.6.0_dont_expand_cudnn_path.patch
+    'c99011f1e41bf62ca989c0a572e55c8c657798ab1d99db35c2ca8ccf93734490',  # TensorFlow-1.6.0_dont_expand_cuda_path.patch
+    '0ad82aab981721b1ed4881106d9accd43fe38b863ec147f6a16b278605012099',  # TensorFlow-1.6.0_remove-msse-hardcoding.patch
+]
+
+builddependencies = [
+    ('Bazel', '0.11.0'),
+    ('wheel', '0.30.0', '-Python-%(pyver)s'),
+]
+dependencies = [
+    ('Python', '3.6.4'),
+    ('cuDNN', '7.0.5', '-CUDA-%s' % cuda_ver, True),
+]
+
+with_mkl_dnn = False
+
+cuda_compute_capabilities = ['3.0', '3.2', '3.5', '3.7', '6.0', '6.1']
```

Comment:
@smoors Please use the same list we have in `TensorFlow-1.5.0-goolfc-2017b-Python-3.6.3.eb` (cc @akesandgren)?